.circleci/config.yml

```diff
@@ -86,26 +86,54 @@ jobs:
           command: make docs
       - run:
           name: "MkDocs"
-          command: make build-mkdocs
+          # TODO(spencer): Update with new url
+          command: MK_DOCS_SITE_URL="https://head.coda.io/developers/temp-developer-docs/packs/latest/" make build-mkdocs
```

Comment:
This is just bringing it up to date with `main` before you update the url? Aka we should just be rubber-stamping this?

Reply:
yeah i just noticed it's out of date on staging since the workflow hasnt been run in a while